### DISCUSSION TITLE
I would like to know whether strimzi-kafka-operator 0.18.0 is compatible with k8s 1.22 version. I would be very happy to hear the reply.

### DISCUSSION THREAD

**Discussion by xiaoxiongxyy** [strimzi/strimzi-kafka-operator#10841] [Q&A]
I would like to know whether strimzi-kafka-operator 0.18.0 is compatible with k8s 1.22 version. I would be very happy to hear the reply.

**Comment by xiaoxiongxyy** [comment]
What version do we need to upgrade from strimzi-kafka-operator 0.18.0 to for k8s 1.22? The lower the better.

**Reply by scholzj** [reply]
You are asking about versions that are many years old. You might need to find out what works and doesn't work by simply trying it out as nobody remembers anymore.

**Reply by xiaoxiongxyy** [reply]
> You are asking about versions that are many years old. You might need to find out what works and doesn't work by simply trying it out as nobody remembers anymore.

ok thanks！
